webui-src/assets/index.html

```diff
@@ -9,19 +10,22 @@
     <link href="/data/retroshare.svg" rel="icon" data-n-head="true" type="image/svg">
     <script src="app.js"></script>
 </head>
+
 <body onload="load_ui();">
     <div id="main">
-	<h3>
-	Javascript not found!
-	</h3>
-    <script type="text/javascript">
-        function load_ui(){
-            var main = require('main');
-        }
-    </script>
-    <noscript>Sorry but JavaScript is needed here! Please enable it in your browser.</noscript>
+        <h3>
+            Javascript not found!
+        </h3>
+        <script type="text/javascript">
+            function load_ui() {
+                var main = require('main');
+                document.getElementById('password').focus();
```

Comment:
Why not use the autofocus attribute

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

Reply:
For some reason, autofocus is not working.
So, I tried to do it with JavaScript 😅

Reply:
Ok it's better to do this in the oncreate lifecycle method

Reply:
Fixed it, Thanks @rottencandy